src/OnChain.ts

```diff
@@ -91,7 +92,7 @@ export const OnChainMixin = (superclass) => class extends superclass {
 
       const payeeUser = await this.tentativelyGetPayeeUser({address})
 
-      // TODO? isSendAll affects this path? shows fee=0
+      /// TODO? isSendAll affects this path? shows fee=0
```

Comment:
fee is 0 because this is an internal ledger transaction. but I think it still affect this path. when sendAll = true, the amount is basically not taken into account.